### DISCUSSION TITLE
Lifove bible 앱을 활성화 시킬 수 있나요?

### DISCUSSION THREAD

**Discussion by kim-sang-ho** [community/community#110265] [New to GitHub]
현재 Lifove bible 앱은 업데이트가 중단 되어 사용이 어렵습니다.
이 앱을 업데이트 해줄 개발자를 찿고 있습니다

**Comment by queenofcorgis** [comment]
Hi @kim-sang-ho , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and  locking any posts in languages other than English for now, including this one.
